### ISSUE TITLE
Delete posts with empty title

### ISSUE THREAD

**Issue by mamadaliev** [Reckue/post-ui#101]
Please fix creating posts with empty title on edit page.

**Comment by Hardelele** [comment]
Yes, thank you. Long ago want to create that issue, but always forget)

**Comment by Hardelele** [comment]
If user leave post-edit page with empty title, then cause alert message with 2 options: delete post or add title.
Have to create subtask for alert message component.

**Comment by Hardelele** [comment]
You need to use popup-modal component. If you dont have some functionons in popup modal you could write additional info about new features and implement it. Look this issue #132

**Comment by Hardelele** [comment]
If user dont write post title in popup modal you ask him to do this. 
 #
If he closes popup you remove empty post. 
 #
In future we will not removing post, just chage title to default. But yet we dont need to complex this logic, so just delete post.

**Comment by Hardelele** [comment]
Popup modal should be showed up when you leave post-edit page.
